utils/package.json

```diff
@@ -1,37 +1,37 @@
 {
-  "name": "rainbow-bridge-utils",
-  "version": "1.0.0",
-  "description": "Rainbow Bridge utils and config",
-  "main": "index.js",
-  "author": "Near Inc. <hello@near.org>",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/near/rainbow-bridge.git"
-  },
-  "license": "MIT",
-  "keywords": [
-    "nearprotocol",
-    "decentralized",
-    "blockchain",
-    "ethereum",
-    "bridge"
-  ],
-  "dependencies": {
-    "bn.js": "^5.1.3",
-    "bs58": "^4.0.1",
-    "bsert": "^0.0.10",
-    "change-case": "^4.1.1",
-    "configstore": "^5.0.1",
-    "eth-object": "near/eth-object#54e03b8aac8208cf724e206d49ffb8bdd30451d7",
-    "eth-util-lite": "near/eth-util-lite#master",
-    "lodash": "^4.17.20",
-    "near-api-js": "^0.26.0",
-    "web3": "=1.2.6"
-  },
-  "devDependencies": {
-    "standard": "*"
-  },
-  "scripts": {
-    "standard": "standard"
-  }
+    "name": "rainbow-bridge-utils",
+    "version": "1.0.0",
+    "description": "Rainbow Bridge utils and config",
+    "main": "index.js",
+    "author": "Near Inc. <hello@near.org>",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/near/rainbow-bridge.git"
+    },
+    "license": "MIT",
+    "keywords": [
+        "nearprotocol",
+        "decentralized",
+        "blockchain",
+        "ethereum",
+        "bridge"
+    ],
+    "dependencies": {
+        "bn.js": "^5.1.3",
+        "bs58": "^4.0.1",
+        "bsert": "^0.0.10",
+        "change-case": "^4.1.1",
+        "configstore": "^5.0.1",
+        "eth-object": "near/eth-object#54e03b8aac8208cf724e206d49ffb8bdd30451d7",
+        "eth-util-lite": "near/eth-util-lite#master",
+        "lodash": "^4.17.20",
+        "near-api-js": "^0.26.0",
+        "web3": "=1.2.6"
+    },
+    "devDependencies": {
+        "standard": "*"
+    },
+    "scripts": {
+        "standard": "standard"
+    }
```

Comment:
?

Reply:
Can we also enable linter for json-files?

Reply:
I think we should do, for now I'll change my current json formatter to use 2 space indentation, since this seems to be what you are using.